kernel/build/build_rpm

```diff
@@ -84,6 +84,19 @@ if [ "$THROWAWAY" = false ] ; then
     # push binaries to chacra
     find ~/rpmbuild/ | egrep '\.rpm$' | $VENV/chacractl binary ${chacra_flags} create ${chacra_endpoint}/$ARCH/
 
+    # write json file with build info
+    cat > $WORKSPACE/repo-extra.json << EOF
+{
+    "version":"$kernelrelease",
```

Comment:
@idryomov @andrewschoen I'm a bit confused - we do still need `package_manager_version` - but this PR doesn't contain it. If we also need `version` (and it sounds like we do), we can use that too.